pkgs/development/python-modules/sip/feat-parallel-configure.patch

```diff
@@ -0,0 +1,275 @@
+From 3b86b3489dcbdcef3a83312cd168e05458a0d0ad Mon Sep 17 00:00:00 2001
```

Comment:
Can we try to upstream this or at least parts?

Reply:
[upstream says no](https://www.riverbankcomputing.com/pipermail/pyqt/2022-June/044717.html)

> Thanks but I’m not looking at optimisations until all the C code has 
> been replaced by Python code.
> 
> Phil